hw11/func.py

```diff
@@ -0,0 +1,63 @@
+#  работа с функциями в Jupyter
+
+from sympy import *
+from sympy.plotting import plot
+from sympy import Symbol
+import matplotlib.pyplot as plt
+import numpy as np
+
+# Требуется:
+# Найти точки пересечения
+# Построить графики функций в одной системе координат
+# Построить график обобщенной функции
+
+x = Symbol("x")
+
+# f(x) = x^3 - 50x и g(x) = -x^4 + 88x^2 - 241
+
+fx = x**3 - 50*x
+
+gx = -x**4 + 88*x**2 - 241
+g1 = gx
+# dx = fx - gx
+
+dx = x**3 - 50*x - (-x**4 + 88*x**2 - 241)
+d1 = dx
+# dx = 𝑥4+𝑥3−88𝑥2−50𝑥+241
```

Comment:
```suggestion
fx = x**3 - 50*x
gx = -x**4 + 88*x**2 - 241
dx = fx - gx
```
Вот и все. зачем столько промежуточных выражений?